.github/workflows/ci.yml

```diff
@@ -30,9 +30,32 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 12.x
-      - run: yarn install --frozen-lockfile --ignore-engines
+      - run: yarn install --ignore-engines
       - run: yarn add --dev typescript@~${{ matrix.ts-version }} --ignore-engines
       - run: yarn test-typings
+  test-db2:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [10, 12, 14]
+    name: DB2 (Node ${{ matrix.node-version }})
+    runs-on: ubuntu-latest
+    env:
+      DIALECT: db2
+      SEQ_DB: testdb
+      SEQ_USER: db2inst1
+      SEQ_PW: password
+      SEQ_TEST_CLEANUP_TIMEOUT: 1200000
+      SEQ_PORT: 50000
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - run: npm run start-db2
+      - name: Tests
+        run: npm run test-db2    
```

Comment:
I think for the sake of consistency, I would try to make it consistent with the other dialects (as in split unit + integration tests)

Reply:
Since I don't see any service declaration here, against what is the test actually connecting?

Reply:
I would have somehow expected to see us spawning a copy of ibmcom/db2-amd64:11.5.6.0a maybe?

Reply:
> I think for the sake of consistency, I would try to make it consistent with the other dialects (as in split unit + integration tests)

Done.

Reply:
> I would have somehow expected to see us spawning a copy of ibmcom/db2-amd64:11.5.6.0a maybe?

"npm run start-db2" command in CI actually downloads the ibmdb DB2 container and against the same tests run.